dkc/core/rest/folder.py

```diff
@@ -379,3 +380,8 @@ def public(self, request, pk=None):
         tree.public = serializer.validated_data['public']
         tree.save()
         return Response(serializer.data, status=200)
+
+    def destroy(self, request, *args, **kwargs):
```

Comment:
Could you place this method definition further up, above the `@action` methods? I think that'd be an easier code layout for understanding this `ViewSet`